tests/conftest.py

```diff
@@ -296,7 +298,7 @@ def load_firmware(tests_setup: Setup, request):
 
     # Reboot drive
     client.exposed_turn_off_ps()
-    time.sleep(1)
+    time.sleep(SLEEP_BETWEEN_POWER_CYCLE_S)
```

Comment:
Can you indicate why this was necessary?

Reply:
I'm not sure, it was introduced [here](https://github.com/ingeniamc/ingeniamotion/commit/927da56d62e7b3a98a9830fb8825eaaeede3948c).

Reply:
I think we wanted to make sure that the power supply was on before the tests were run.